File: src/main/java/com/enonic/lib/react4xp/AppHelper.java
package com.enonic.lib.react4xp;

import com.enonic.xp.home.HomeDir;
import com.enonic.xp.server.RunMode;

public class AppHelper
{
  public boolean isDevMode()
  {
    return RunMode.get() == RunMode.DEV;
  }

  public String getXpHome()
  {
      return HomeDir.get().toString();
  }
}

File: src/main/resources/lib/enonic/react4xp/xp/appHelper.ts
const helper = __.newBean<{
	isDevMode: () => boolean
	getXpHome: () => string
}>('com.enonic.lib.react4xp.AppHelper');
export const IS_DEV_MODE = helper.isDevMode();
export const IS_PROD_MODE = !IS_DEV_MODE;

const XP_HOME = helper.getXpHome();

export function getXpHomeDirPathAbsolute() {
	return XP_HOME;
}
